test/unit/specs/components/common/TmBalance.spec.js

```diff
@@ -31,12 +36,10 @@ describe(`TmBalance`, () => {
   it(`shows correct stats depending on props`, () => {
     expect(wrapper.contains(`.unbonded-atoms`)).toBe(true)
 
-    expect(wrapper.contains(`.total-earnings`)).toBe(false)
```

Comment:
why did you remove those checks?

Reply:
they required the props to be set

Reply:
and it was already tested on line 41

Reply:
hmm, I think not.
```
expect(wrapper.contains(`.total-earnings`)).toBe(false)
```
is the negative test (it not showing yet)

```
expect(wrapper.contains(`.total-earnings`)).toBe(true)
```
is the positive test (now it is showing)

Maybe set the `totalEarnings` to 0 before?

Reply:
👍